docs/topics/settings.txt

```diff
@@ -125,7 +125,7 @@ importing individual settings is not possible::
 
     from django.conf.settings import DEBUG  # This won't work.
 
-Also note that your code should *not* import from either ``global_settings`` or
+Also, note that your code should *not* import from either ``global_settings`` or
```

Comment:
Given that the preceding point is `Note` I think it's correct to `Also note`, rather than `Also, note`. Given that this would require a wrapping change, I don't think it's worth it.